### ISSUE TITLE
qhull_a.h(106): error: this declaration may not have extern "C" linkage (Intel compiler)

### ISSUE THREAD

**Issue by slayoo** [gnudatalanguage/gdl#229]
As captured here: https://travis-ci.org/slayoo/gdl/jobs/368865358
compilation with QHULL on Linux with Intel compiler fails with:
```
gdl/src/math_fun_jmg.cpp(64):
/usr/include/libqhull/qhull_a.h(106): error: this declaration may not have extern "C" linkage
  template <typename T>
```

**Comment by GillesDuvert** [comment]
Actually QHULL is **NOT USED** in GDL and the cmake script should, for the time being, **put PL_HAVE_QHULL to false** whaterer the presence of the qhull library.
Qhull will be used when somebody writes the qhull and qgrid3 procedure & function.
TRIANGULATE is done inside GDL using an opensource version of the historical code used by IDL, as the syntax of triangulate and its dependecies impose it.

**Comment by GillesDuvert** [comment]
closed by #257
